modules/logging-loki-retention.adoc

```diff
@@ -8,12 +8,17 @@
 
 With Logging version 5.6 and higher, you can configure retention policies based on log streams. Rules for these may be set globally, per tenant, or both. If you configure both, tenant rules apply before global rules.
 
+[IMPORTANT]
```

Comment:
@smunje1 - What do you think about making this a snippet, so that we can place it in a few relevant places? I think perhaps this notice should be a bit placed somewhere in the installation section, as well.

Reply:
Hello @libander,

> What do you think about making this a snippet, so that we can place it in a few relevant places?

Done.

> I think perhaps this notice should be a bit placed somewhere in the installation section, as well.

Thanks for your suggestion. However, it seems to be out of scope for this Jira's PR. 

PS: Please let me know if there is any Jira that you want me to look into for the inclusion of this snippet. Thanks!